src/Community.VisualStudio.Toolkit.14.0/Community.VisualStudio.Toolkit.14.0.csproj

```diff
@@ -11,6 +11,7 @@
   <ItemGroup>
     <PackageReference Include="Madskristensen.VisualStudio.SDK" Version="14.0.83" IncludeAssets="All" PrivateAssets="None" />
     <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0" PrivateAssets="All" />
+    <PackageReference Include="Microsoft.VisualStudio.Shell.Interop.12.1.DesignTime" Version="12.1.30329" />
```

Comment:
For some reason both the 14 and 16 versions of the `Microsoft.VisualStudio.SDK` don't reference this package whereas the v15 does.  This is necessary to bring in the `IVsSharedAssetsProject` interface type.